bravado/response.py

```diff
@@ -93,3 +94,20 @@ def status_code(self):
 
     def json(self, **kwargs):
         return self._delegate.json(**kwargs)
+
+
+class FidoLibResponseAdapter(ResponseLike):
```

Comment:
yeah, basically i wanted to keep both the namings consistent. I am changing `RequestsLibResponse..` to `RequestsResponse..` as well.